packages/effect/src/ConfigError.ts

```diff
@@ -2,8 +2,42 @@
  * @since 2.0.0
  */
 import type * as Cause from "./Cause.js"
+import type { And, InvalidData, MissingData, Or, SourceUnavailable, Unsupported } from "./internal/configError.js"
 import * as internal from "./internal/configError.js"
 
+export {
```

Comment:
We can't re-export from internal without explicit typing, files in internal don't appear in the compiled `.d.ts`